examples/testplans/release/release.json

```diff
@@ -37,7 +37,7 @@
 	 },
          {
 	 "name": "Configure Read The Docs",
-         "description": "Visit the link below:\n\nhttps://readthedocs.org/dashboard/avocado-framework/edit/\n\n 1) Click in *Versions*. In *Choose Active Versions*, find the version you're releasing and check the *Active* option. *Submit*;\n 2) Click in *Versions* again. In *Default Version*, select the new version you're releasing. *Submit*."
+         "description": "Visit the link below:\n\nhttps://readthedocs.org/dashboard/avocado-framework/edit/\n\n 1) Click in *Versions*. In *Choose Active Versions*, find the version you're releasing and check the *Active* option. *Submit*;\n 2) Click in *Admin*. In *Default Version*, select the new version you're releasing. *Submit*."
```

Comment:
I tried to follow the instructions and I think that still it's not correct. What about this?
1) Click in *Versions*. Under *Activate a version*, find the version you're releasing and check the *Activate* option. 
2) Click in *Admin*. Under *Advanced Settings* in *Default Version*, select the new version you're releasing. *Save*.

Reply:
Yep, looks better.  Sending an update.

Reply:
I ended up with:

1) Click in *Versions*. Under *Activate a Version*, find the version you're releasing and click the *Activate* button;
2) Click in *Admin*. Under *Advanced Settings*/*Global settings*/*Default Version*, select the new version you're releasing. Click the *Save* button at the bottom of the page."